docs/tasks/configure-pod-container/configure-volume-storage.md

```diff
@@ -0,0 +1,107 @@
+---
+---
+
+{% capture overview %}
+
+This page shows how to configure a Pod to use a Volume for storage.
+
+A Container's file system lives only as long as the Container does, so when a
+Container terminates and restarts, changes to the filesystem are lost. For more
+consistent storage that lasts for the life of a Pod, you can use a
+[Volume](/docs/user-guide/volumes). This is especially important for stateful
+applications, such as key-value stores and databases. For example, Redis is a
+key-value cache and store.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Configuring a volume for a Pod
+
+In this exercise, you create a Pod that runs one Container. The Pod has a
+volume that lasts for the life of the Pod, even if the container terminates and
+restarts. Here is the configuration file for the Pod:
```

Comment:
WDYT of:

In this exercise, you create a Pod that runs one Container. The Pod has an` Volume of type EmptyDir`, which lasts for the life of the Pod, even if the container terminates and restarts.  `Some Volumes, such as PersistentVolumeClaims, even continue to exist outside the lifetime of the Pod.`